README.md

```diff
@@ -81,7 +81,7 @@ The map of containers consists of the name of the container mapped to the contai
 	* `device` (array) Add host devices.
 	* `dns` (array)
 	* `entrypoint` (string)
-	* `env` (array)
+	* `env` (array/mapping) `env` can be declared as a mapping where each `key`-`value` will be translated to a string `"key=value"`.
```

Comment:
Maybe with something like:
* `env` (array/mapping) It can be declared as a string array with `"key=value"` format or a mapping where each `key: value` will be translated to the corresponding `"key=value"` string.

And for the example, maybe we could add a note below indicating that env could be also declared as:
```
env:
  MYSQL_ROOT_PASSWORD: mysecretpassword
```
I don't think it would be necessary to copy again the whole example.

In addition, if this PR is merged, the other one for the advanced features of YAML could be updated to show another example using this mapping-env with anchors behaving as variables.